sys/include/pipe.h

```diff
@@ -32,8 +32,8 @@
  * @author      René Kijewski <rene.kijewski@fu-berlin.de>
  */
 
-#ifndef __PIPE__H
-#define __PIPE__H
+#ifndef PIPE__H
+#define PIPE__H
 
```

Comment:
could you change also the `#endif` in [L122](https://github.com/rajma996/RIOT/blob/master/sys/include/pipe.h#L122) to `#endif /* PIPE__H */`?